libcloud/loadbalancer/providers.py

```diff
@@ -42,6 +42,9 @@
     ('libcloud.loadbalancer.drivers.softlayer', 'SoftlayerLBDriver'),
     Provider.DIMENSIONDATA:
     ('libcloud.loadbalancer.drivers.dimensiondata', 'DimensionDataLBDriver'),
+    Provider.SLB:
```

Comment:
Same here, probably a good idea to be explicit and add `ALIYUN_` prefix to every provider constant.